WordPress/src/main/java/org/wordpress/android/ui/reader/services/search/ReaderSearchServiceStarter.java

```diff
@@ -0,0 +1,54 @@
+package org.wordpress.android.ui.reader.services.search;
+
+import android.app.job.JobInfo;
+import android.app.job.JobScheduler;
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.os.PersistableBundle;
+import android.support.annotation.NonNull;
+
+import org.wordpress.android.util.AppLog;
+
+/**
+ * service which searches for reader posts on wordpress.com
+ */
+
+public class ReaderSearchServiceStarter {
+    private static final int JOB_READER_SEARCH_SERVICE_ID = 5000;
+    public static final String ARG_QUERY = "query";
+    public static final String ARG_OFFSET = "offset";
+
+    public static void startService(Context context, @NonNull String query, int offset) {
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
+            Intent intent = new Intent(context, ReaderSearchService.class);
+            intent.putExtra(ARG_QUERY, query);
+            intent.putExtra(ARG_OFFSET, offset);
+            context.startService(intent);
+        } else {
+            // schedule the JobService here for API >= 26. The JobScheduler is available since API 21, but
+            // it's preferrable to use it only since enforcement in API 26 to not break any old behavior
```

Comment:
Sorry to be so picky, but just noticed the typo here and in the other services - should be "preferable" rather than "preferrable"

Reply:
:D no problem! always keen to learn how to use English properly 👍  addressed in ba7ac92